{
  try {
    process();
  }
 catch (  Exception x) {
    anyToFailure(x);
  }
}
